src/relation/transforms.rs

```diff
@@ -2,6 +2,8 @@
 //!
 
 use super::{Join, Map, Reduce, Relation, Set, Table, Values, Variant as _};
+// use crate::data_type::product::Error;
+use crate::data_type::value::Variant;
```

Comment:
Do you explicitly use `Variant` or do you just need the method of the trait, in this case `use Variant as _`

Reply:
`data_type` is already imported, please always factor the crate uses